Example/Tests/Tests.m

```diff
@@ -934,8 +981,138 @@ - (ADBMediaObject *)createWithProperties:(NSDictionary *)properties andEventType
+                                                                                                                        @"bitrate" : @500000,
+                                                                                                                        @"startup_time" : @2,
+                                                                                                                        @"fps" : @24,
+                                                                                                                        @"dropped_frames" : @10
+                                                                                                                        } context:@{}
+                                                                     integrations:@{}];
+                
+                [integration track:payload];
+                [verify(mockPlaybackDelegate) createAndUpdateQOSObject:@{
+                                                                 @"bitrate" : @500000,
+                                                                 @"startup_time" : @2,
+                                                                 @"fps" : @24,
+                                                                 @"dropped_frames" : @10
+                                                                 }];
+            });
+            
+            it(@"tracks Quality of Service event without properties", ^{
+                SEGMockADBMediaHeartbeatFactory *mockADBMediaHeartbeatFactory = [[SEGMockADBMediaHeartbeatFactory alloc] init];
+                mockADBMediaHeartbeatFactory.ADBMediaHeartbeat = mockADBMediaHeartbeat;
+                
+                SEGMockPlaybackDelegateFactory *mockPlaybackDelegateFactory = [[SEGMockPlaybackDelegateFactory alloc] init];
+                mockPlaybackDelegateFactory.playbackDelegate = mockPlaybackDelegate;
+                
+                SEGTrackPayload *payload = [[SEGTrackPayload alloc] initWithEvent:@"Video Quality Updated" properties:@{} context:@{}
+                                                                     integrations:@{}];
+                
+                [integration track:payload];
+                [verify(mockPlaybackDelegate) createAndUpdateQOSObject:@{}];
+            });
+            
+        });
+
+    });
+});
+
+describe(@"SEGPlaybackDelegate", ^{
+    __block SEGPlaybackDelegate *playbackDelegate;
+    __block SEGPlaybackDelegate *mockDelegate;
+    __block SEGMockPlaybackDelegateFactory *mockPlaybackDelegateFactory = [[SEGMockPlaybackDelegateFactory alloc] init];
 
+    beforeEach(^{
+        mockPlaybackDelegateFactory.playbackDelegate = mockDelegate;
+        mockDelegate = mock([SEGPlaybackDelegate class]);
+
+        playbackDelegate = [[SEGPlaybackDelegate alloc] initWithDelegate:mockDelegate];
     });
 
+    describe(@"SEGPlaybackDelegate", ^{
```

Comment:
seems like an unnecessary level of nesting (it's already under`SEGPlaybackDelegate` from line 1035).